### ISSUE TITLE
A11y_AzureDataStudio_New deployment - SQL Server on Windows_Screen Reader: Narrator is not announcing install location field in specify SQL server install location page

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#18591]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1.3.1; #Win32; #AINotInScope; #Info and Relationships; #Narrator; #NVDA;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on Narrator
2. Launch Azure Data Studio.
3. Select the Connections icon (if not already selected) from left bar.
4. Click on the ellipses '...' and select "New Deployment"
5. In the Deployment options, select "SQL Server on Windows" card.
5. From the dropdown for Version under Options, select from the available options.
6. Click the "Select" button  
7. This will start the setup installer for SQL Server setup. 
8. Follow the instructions through the setup process to install a SQL Server on Windows.
9. Verify install location field in specify SQL server install location page is announced by narrator or not 

### Actual Result:
Narrator is not announcing install location field in specify SQL server install location page

### Expected Result:
Narrator should announce install location field in specify SQL server install location page

### Attachment:
**Narrator Attachment:**

![A11y_AzureDataStudio_New deployment - SQL Server on Windows_Screen Reader_Narrator](https://user-images.githubusercontent.com/100189407/156150703-6c27fb50-4d26-4c1c-b0e1-c50529e115ad.gif)

**NVDA Attachment:**

![A11y_AzureDataStudio_New deployment - SQL Server on Windows_Screen Reader_NVDA (2)](https://user-images.githubusercontent.com/100189407/156151248-90a7ead1-e0d7-4a55-a24e-a39464860ea0.png)

**Comment by alanrenmsft** [comment]
the install SQL server feature you tested is not a part of AzureDataStudio. so closing it, let me know if you don't know the owner of the sql server installer.

**Comment by VijayaBharathyA** [comment]
@alanrenmsft Could you please let us know if we need to route this bug to the sql server installer team? If yes, could you please let us know the vso and owner's detaills?  and also can you resolve that bug, because bug can be closed by trusted tester only

**Comment by VijayaBharathyA** [comment]
@alanrenmsft Could you please route it to the appropriate owner or team and reactivate the bug.
**Note:** "Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"

**Comment by VijayaBharathyA** [comment]
Verified issue in below Environment:
Version: 1.41.1 (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Issue is fixed
![image](https://user-images.githubusercontent.com/100189407/216271583-eb3dc7e7-c9e7-4608-bbf4-1fc819688242.png)
